### PR TITLE
docs(): various refinements

### PR DESCRIPTION
**Pull request by andrii-bodnar**

### CODE REVIEW

File: packages/cli/src/lingui-extract.ts [hunk 1]
```diff
@@ -127,7 +127,7 @@ if (require.main === module) {
     hasErrors = true
     const msg =
       "--babelOptions option is deprecated." +
-      " Please set extractBabelOptions in configuration https://lingui.js.org/ref/conf.html#extractBabelOptions"
+      " Please set extractBabelOptions in configuration https://lingui.dev/ref/conf#extractBabelOptions"
```
Comment (Martin005) [inline packages/cli/src/lingui-extract.ts:130]: This anchor doesn't work as anchors are case sensitive. Please update `#extractBabelOptions` to be in lower-case

File: packages/core/README.md [hunk 2]
```diff
@@ -25,9 +25,9 @@ This package is licensed under [MIT][license] license.
 
 [license]: https://github.com/lingui/js-lingui/blob/main/LICENSE
 [linguijs]: https://github.com/lingui/js-lingui
-[documentation]: https://lingui.js.org/
-[tutorial]: https://lingui.js.org/tutorials/js.html
-[reference]: https://lingui.js.org/ref/core.html
+[documentation]: https://lingui.dev
+[tutorial]: https://lingui.dev/tutorials/js
```
Comment (Martin005) [inline packages/core/README.md:29]: This was moved to `https://lingui.dev/tutorials/javascript`

**Review by Martin005 — CHANGES_REQUESTED** [review]
There are still a few places in the code that point to `lingui.js.org`. Could you please update them as well? 🙂

**Comment by andrii-bodnar** [comment]
There are still a few `lingui.js.org` links in the `CHANGELOG.md` files, I think it will be better not to touch these files and keep this history as is.

**Review by Martin005 — APPROVED** [review]

**Pull request merged**